src/hamster/widgets/facttree.py

```diff
@@ -45,6 +46,20 @@ def __init__(self):
 
         self.width = 50 # Simon says
 
+class TotalFact(Fact):
```

Comment:
I find it awkward that this class inherits from "Fact" given that it "_doesn't have a meaningful  start and an end_".

Reply:
Yeah, I made it a subclass of Fact so I didn't have to change some of the methods in the `FactTree` class. Some of the methods in this class were already dealing with arrays or instanced of the `Fact` class. Making the Total row a Fact made it easier (in my opinion) to introduce the changes pursued by this PR. If you prefer, I can make the `TotalFact` not to extend from `Fact`, but the code in `FactTree` will become messier (again, in my opinion).

Reply:
Sure, it was easier this way :grin:. I wondered if it'd be possible to separate out the functionality that "TotalFact" needs from "Fact" into another base class, and then have them both inherit from that new base class. But I haven't looked deep enough to be certain that that would be feasible.

I guess I'd be fine with just a comment with a FIXME saying that one day, the class hierachy needs to be cleaned up.

Reply:
I'll definitely will add a comment. I was also thinking about creating a superclass for both classes, but I didn't want to mess around too much with the code in my first PR! :)

Reply:
I really understand that you don't want to mess around with a lot of code in the first PR, @esteban-aliverti :)